asmcli/lib/validate.sh

```diff
@@ -330,7 +330,11 @@ exit_if_cluster_unlabeled() {
   local LABELS; LABELS="$(get_cluster_labels)";
   local REQUIRED; REQUIRED="$(mesh_id_label)";
   local NOTFOUND; NOTFOUND="$(find_missing_strings "${REQUIRED}" "${LABELS}")"
-
+  local DISABLE_UI; DISABLE_UI="$(context_get-option "DISABLE_UI")"
```

Comment:
I realize I should have caught this sooner, but we should do this first thing in the function since the functions that get called above it might reach out to external APIs to get information.